xbmc/cores/VideoPlayer/DVDInputStreams/DVDFactoryInputStream.cpp

```diff
@@ -137,12 +137,15 @@ CDVDInputStream* CDVDFactoryInputStream::CreateInputStream(IVideoPlayer* pPlayer
     if (fileitem.IsType(".m3u8"))
       return new CDVDInputStreamFFmpeg(fileitem);
 
+    //why are members set ath this global place? Disabled (mapfau)
+    /*
     if (fileitem.ContentLookup())
     {
       // request header
       fileitem.SetMimeType("");
       fileitem.FillInMimeType();
     }
+    */
 
```

Comment:
the code above seems not to be at the right place, should be moved to the implementation class where its really thought for.